packages/nextjs/src/tools/codegen/extract-files.ts

```diff
@@ -9,17 +9,18 @@ import {
 import { constants } from '@sitecore-content-sdk/core';
 import { SitecoreConfig } from '@sitecore-content-sdk/core/config';
 import { fetchBearerToken } from '@sitecore-content-sdk/core/tools';
+import path from 'path';
 
-export type ExtractComponentsConfig = {
+export type ExtractFilesConfig = {
   scConfig: SitecoreConfig;
   componentMapPath?: string;
 };
 
 /**
  * Extracts components from the app folder and sends them to XMCloud.
- * @param {ExtractComponentsConfig} args - Config for components extraction
+ * @param {ExtractFilesConfig} args - Config for components extraction
  */
-export const extractComponents = (args: ExtractComponentsConfig) => {
+export const extractFiles = (args: ExtractFilesConfig) => {
```

Comment:
RE: consent. The flow is not ready on product side yet, consent process will be implemented seprately